### ISSUE TITLE
how to style the countdown timer like on your homepage?

### ISSUE THREAD

**Issue by kinganu** [hilios/jQuery.countdown#259]
http://hilios.github.io/jQuery.countdown/ 

Pardon my earlier comment, I got the UTC timer working using your example and the proper imports, moment-js and moment-js-with-data.  Happy to have this little snippet working, thanks and sorry for earlier frustration.  Cheers

**Comment by JamesNock** [comment]
No offence but it's really not the developer's job to teach you javascript or how to integrate code that they have taken the time to write and make free of charge to you. If you're not sure how to use it then why not pay someone that does.

**Comment by FagnerMartinsBrack** [comment]
@kinganu This is open-source. You should fix the problem by learning and opening Pull Requests to improve the documentation. The only reason why you're frustrated is that somebody else saw the problems and didn't fix it, just like what you're doing right now.

My suggestion would be to take the time to learn how to use it, find the problems and then open Pull Requests to improve the documentation.

**Comment by kinganu** [comment]
youre right guys, my bad.  I figured it out, had to get all the imports in the head, in proper order, then wrap the JS part in your timezone example in script tags on my index page.  now just wondering how to style it like the one on the homepage.  ill search for how, unless anyone knows some style code around.  thanks

**Comment by bafsar** [comment]
Hi kinganu,

To find that style, you can use page source. In page source, you have to find template script which has id is main-example-template. Also, the script code which came after template.

Remains css codes. You can reach css code that you need in main.css file.

**Comment by kinganu** [comment]
ok I looked at that, unfortunately its a ridiculous amount of code just to style this countdown timer, 252 lines of html and css.  seriously that is overkill for something this small.  wish the example on  http://hilios.github.io/jQuery.countdown/examples/timezone-aware.html could have a minimal css style for it.

**Comment by bafsar** [comment]
If you prefer some other library like this, you can use [FlipClock.js](http://flipclockjs.com/). After a look this library, I gave up to use it in my project. Because it has causes a visual bug in Chrome based browsers on Windows. (I haven't tried on any other OS).

Implementation of FlipClock is quite simple and it seems pretty nice.
